README.md

```diff
@@ -1,18 +1,44 @@
-# Getting Started with Create React App
+# CUBluePrint's New Website 
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Our new and updated website for CUBluePrint!
 
-## Available Scripts
+You can find our live website available at: [link](enter link here thanks)
+
+
+
+## Motivation for Project
+
+In 2021, we decided to build our club a brand new website with an improved UI and a more modern tech stack. The reason for this change was that our old website was built with __[insert outdated technology]__ and was becoming increasingly difficult for our exec team to maintain.
```

Comment:
I'm pretty sure the old [site](https://github.com/cublueprint/cublueprint-website) used Pug, a templating engine taught in COMP2406. I think the main motivation for the revamp was just the outdated design decisions.